examples/mxnet_simple.py

```diff
@@ -0,0 +1,128 @@
+"""
+Optuna example that optimizes multi-layer perceptrons using MX-NET.
+
+In this example, we optimize the validation accuracy of hand-written digit recognition using
+MX-NET and MNIST. We optimize the neural network architecture as well as the optimizer
+configuration. As it is too time consuming to use the whole MNIST dataset, we here use a small
+subset of it.
+
+We have the following two ways to execute this example:
+
+(1) Execute this code directly.
+    $ python mxnet_simple.py
+
+
+(2) Execute through CLI.
+    $ STUDY_NAME=`optuna create-study --storage sqlite:///example.db`
+    $ optuna study optimize mxnet_simple.py objective --n-trials=100 --study $STUDY_NAME \
+      --storage sqlite:///example.db
+
+"""
+
+from __future__ import print_function
+
+
+import mxnet as mx
+import numpy as np
+
+
+N_TRAIN_EXAMPLES = 3000
+N_TEST_EXAMPLES = 1000
+BATCHSIZE = 128
+EPOCH = 10
+
+
+def model_fn(trial):
```

Comment:
Could you change the function name to `create_model` because it seems to be more consistent with `create_optimizer` than `model_fn`.

Reply:
Yeah. Will do.